test/test_biogeochemistry.jl

```diff
@@ -12,6 +12,26 @@ import Oceananigans.Biogeochemistry:
     biogeochemical_advection_scheme, 
     update_biogeochemical_state!
 
+@kernel function update_PAR!(PAR, grid, P, t, bgc) 
+    i, j = @index(Global, NTuple) 
+    
+    surface_PAR = bgc.surface_PAR(t)
+    
+    z = grid.zᵃᵃᶜ[grid.Nz]
```

Comment:
This only works on rectilinear grids. Use

```suggestion
    z = znode(Center(), Center(), Center(), i, j, grid.Nz, grid)
```

for generality on all grids.